packages/components-native/tsconfig.json

```
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./dist",
```

```suggestion
    "outDir": "./dist/esm",
```
Or update the package.json

---

Good catch! I updated the `package.json`